documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc

```diff
@@ -44,11 +44,14 @@ spec:
       metadata:
         name: redhat-operator-index
       spec:
-        image: infra.5g-deployment.lab:8443/redhat/redhat-operator-index:v4.12-1677697786 #TODO
+        image: infra.5g-deployment.lab:8443/redhat/redhat-operator-index:v4.13 #TODO
```

Comment:
You could use this tag: v4.13-1688429207

built on: GMT: Tuesday, July 4, 2023 12:06:47 AM

Reply:
Great, thanks!